### PR TITLE
Readme - descriptions for Windows container images

### PR DESCRIPTION
**Pull request by PatrickLang**

### CODE REVIEW

**Comment by dleeapho** [comment]
I think we need to call out the Windows SDK images as a separate item in the readme:
•	1.0.0-preview2-windowsservercore-sdk, windowsservercore
•	1.0.0-preview2-nanoserver-sdk, nanoserver

**Comment by MichaelSimons** [comment]
Is there a reason this PR was made against my private fork vs dotnet/dotnet-docker?

File: 1.0.0/README.md [hunk 1]
```diff
@@ -60,13 +62,23 @@ $ docker build -t my-dotnet-app .
 $ docker run -it --rm --name my-running-app my-dotnet-app
 ```
 
+### `microsoft/dotnet:<version>-core`
```
Comment (MichaelSimons) [inline 1.0.0/README.md]: What was the reason for reordering Core before Core-deps?  What are your thoughts on matching the order here with the tag list above.  I don't have much of an opinion on what the order is, but I think it is beneficial to have the order match.

Comment (dleeapho) [inline 1.0.0/README.md]: I do prefer the Core-before-Core-deps order, it seems better to progress from SDK->Core->Core-deps mirroring the progression of [App Types docs](https://dotnet.github.io/docs/core-concepts/app-types.html#self-contained-application). But yes then the tags need to be reordered.

File: 1.0.0/README.md [hunk 2]
```diff
@@ -26,10 +26,12 @@ This project is part of .NET Core command-line (CLI) tools. You can find samples
 -       [`1.0.0-rc2-core` (*1.0.0-rc2/core/Dockerfile*)](https://github.com/dotnet/dotnet-docker/blob/master/1.0.0-rc2/core/Dockerfile)
 -       [`1.0.0-core-deps`, `core-deps` (*1.0.0/core-deps/Dockerfile*)](https://github.com/dotnet/dotnet-docker/blob/master/1.0.0/core-deps/Dockerfile)
 -       [`1.0.0-core`, `core` (*1.0.0/core/Dockerfile*)](https://github.com/dotnet/dotnet-docker/blob/master/1.0.0/core/Dockerfile)
+-       [`1.0.0-windowsservercore-core`, `windowsservercore-core` (*1.0.0/windowsservercore/core/Dockerfile*)](https://github.com/dotnet/dotnet-docker/blob/master/1.0.0/windowsservercore/core/Dockerfile)
+-       [`1.0.0-nanoserver-core`, `nanoserver-core` (*1.0.0/nanoserver/core/Dockerfile*)](https://github.com/dotnet/dotnet-docker/blob/master/1.0.0/windowsservercore/core/Dockerfile)
```
Comment (naamunds) [inline 1.0.0/README.md]: The hyperlink for the Nano Server Dockerfile is pointing to the Windows Server Core Dockerfile.

**Comment by naamunds** [comment]
In addition to adding the sdk images to the README, we should add the onbuild images for windowsservercore and nanoserver.

File: 1.0.0/README.md [hunk 3]
```diff
@@ -60,13 +62,23 @@ $ docker build -t my-dotnet-app .
 $ docker run -it --rm --name my-running-app my-dotnet-app
 ```
 
+### `microsoft/dotnet:<version>-core`
+
+This image contains only .NET Core (runtime and libraries) and it is optimized for running [portable .NET Core applications](http://dotnet.github.io/docs/core-concepts/app-types.html). If you wish to run self-contained applications, please use the `core-deps` image described above.
```
Comment (naamunds) [inline 1.0.0/README.md]: The http://dotnet.github.io/docs/core-concepts/app-types.html link is now broken. It appears that we are now using https://docs.microsoft.com/en-us/dotnet/articles/core/index as the homepage for the documentation, as http://dotnet.github.io/docs redirects there.

Comment (naamunds) [inline 1.0.0/README.md]: If we're going to move the `core-deps` image to the bottom, then it should say "described below" instead of "described above".

**Comment by PatrickLang** [comment]
Sorry about putting this on the wrong branch. The other PR looks good. Thanks for moving it over.

**Pull request closed without merging**